t/unit/backends/test_redis.py

```diff
@@ -119,66 +118,6 @@ def __init__(self, **kwargs):
             pass
 
 
-class sentinel(object):
```

Comment:
`test_redis.py` was in a weird state - it had some methods added from a future version of the file, but some of the changes were missing and the test was failing. 

I overwrote the whole file with a version I took from official celery 4.1, since we don't use Redis anyway.